addons/nvidia/nvidia.yaml

```diff
@@ -26,19 +26,16 @@ spec:
   chartReference:
     chart: nvidia
     repo: https://mesosphere.github.io/charts/staging
-    version: 0.1.2
+    version: 0.2.0
     values: |
       ---
       nvidia-driver:
         image:
           tag: "418.87.01-centos7"
         resources:
-          limits:
```

Comment:
same as it's clone PR:
@hectorj2f You've done a ton of work on resource parameters. Do you have any issue with this change?

Reply:
see the other thread